contrib/platform/test/com/sun/jna/platform/win32/COM/util/IDispatchTest.java

```diff
@@ -1,16 +1,19 @@
 package com.sun.jna.platform.win32.COM.util;
 
 import com.sun.jna.Pointer;
+import com.sun.jna.platform.win32.COM.util.annotation.ComEventCallback;
+import com.sun.jna.platform.win32.COM.util.annotation.ComInterface;
 import com.sun.jna.platform.win32.COM.util.annotation.ComMethod;
 import com.sun.jna.platform.win32.COM.util.annotation.ComObject;
 import com.sun.jna.platform.win32.COM.util.annotation.ComProperty;
 import com.sun.jna.platform.win32.OaIdl.DISPID;
 import com.sun.jna.platform.win32.Ole32;
 import junit.framework.TestCase;
-import static junit.framework.TestCase.assertTrue;
+import static junit.framework.TestCase.assertFalse;
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
+import static junit.framework.TestCase.assertTrue;
 
```

Comment:
Use:
```java
import static org.junit.Assert.assertTrue;
import static org.junit.Assert.assertFalse;
```
This may not seem like a big deal, but the _org.junit.Assert_ class is the one to use for assertions. _junit.framework_ classes are being deprecated.